goa/Classes/FirebaseHelper.cpp

```diff
@@ -36,9 +36,16 @@ string FirebaseHelper::getLowerCase(string eventName)
     return allLowerCase;
 }
 
-void FirebaseHelper::logCustomEvent(string eventName, string parameter_name, double parameter_value)
+void FirebaseHelper::logCustomEvent(string eventName, string parameter_name_1, string  parameter_value_1, string parameter_name_2, double parameter_value_2)
```

Comment:
Use arrays for string param_1, 2, ...

Reply:
Updated parameter names as having a single array for different parameter type won't work